include set-java in dockerfile

rasheedamir opened the pull request:
It won't fix #5

rasheedamir committed:
Create set_java

rasheedamir committed:
add set jdk in the dockerfile

add set jdk in the dockerfile

stakater-user commented:
@rasheedamir Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins.tools.stackator.com/job/stakater%20docker/job/dockerfile-maven-centos/job/PR-6/1/) has Failed!

rasheedamir commented:
The output fails at:

```
Step 12/15 : RUN $M2/set_java && rm $M2/set_java
 ---> Running in f991395250bc
[91m/bin/sh: /usr/share/maven/bin/set_java: Permission denied
[0mThe command '/bin/sh -c $M2/set_java && rm $M2/set_java' returned a non-zero code: 126
```

rasheedamir committed:
add some RUN commands for testing purpose only

add some RUN commands for testing purpose only
- print contents of a directory
- print who is the current user

stakater-user commented:
@rasheedamir Yikes! You better fix it before anyone else finds out! [Build 2](https://jenkins.tools.stackator.com/job/stakater%20docker/job/dockerfile-maven-centos/job/PR-6/2/) has Failed!

rasheedamir committed:
print environment variables

add print environment variables

stakater-user commented:
@rasheedamir Yikes! You better fix it before anyone else finds out! [Build 3](https://jenkins.tools.stackator.com/job/stakater%20docker/job/dockerfile-maven-centos/job/PR-6/3/) has Failed!

rasheedamir commented:
printenv prints

```
Step 11/18 : RUN printenv
 ---> Running in f08918a0fc04
HOSTNAME=ip-172-20-100-132.us-west-2.compute.internal
APP_ROOT=/opt/app
M2=/usr/share/maven/bin
MAVEN_VERSION=3.5.4
PATH=/usr/share/maven/bin:/opt/app/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
maven.home=/usr/share/maven
PWD=/opt/app
JAVA_HOME=/usr/lib/jvm/java
SHLVL=1
M2_HOME=/usr/share/maven
HOME=/opt/app
container=docker
_=/usr/bin/printenv
```

stakater-user commented:
@rasheedamir Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins.cicd-tools.stackator.com/job/Stakater/job/dockerfile-maven-centos/job/PR-6/1/) has Failed!

stakater-user commented:
@rasheedamir Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins.cicd-tools.stackator.com/job/Stakater%20Docker/job/dockerfile-maven-centos/job/PR-6/1/) has Failed!
